5.x/crud-operation-list-entries.md

```diff
@@ -31,6 +31,16 @@ For views, it uses:
 - ```columns/```
 - ```buttons/```
 
+**NOTE**: If not added by developer, Backpack will append the `model key`(usually **id**) as an hidden column and order it by `DESC`. 
+You can modify this behavior by applying yourself an id order:
+```php
+protected function setupListOperation()
+{
+  $this->crud->orderBy('id', 'asc');
+}
+```
+Note that this is a `query constrain`, not a `table constrain`, so the reset button will not reset this order, it becomes the `base query` for that operation, so any filters etc you add on table will be applied AFTER the `query orders/constrains` you define on your setup operation.
+
```

Comment:
Hmmm I think this is way too soon, at this point the user is just learning how everything works... not the right time to instruct about a quirk, if you ask me.

How above we add a subsection after "[Custom Query](https://backpackforlaravel.com/docs/5.x/crud-operation-list-entries#custom-query-1)" called "Custom Order" where we say everything we have to say about this?